service/service.go

```diff
@@ -29,13 +34,23 @@ type Service struct {
 	// are correctly handled.
 	*onet.ServiceProcessor
 	path string
-	// Count holds the number of calls to 'ClockRequest'
+
+	storage *storage
+}
+
+// storageID reflects the data we're storing - we could store more
+// than one structure.
+const storageID = "main"
+
+// storage is used to save our data.
+type storage struct {
 	Count int
+	sync.Mutex
```

Comment:
yes, but not used - I added a commit to correctly lock/unlock the mutex. The service-methods might be called in parallel, and then the storage.Count might raise a race-condition. Or so `go -race` says...